react/styles.global.css

```diff
@@ -0,0 +1,25 @@
+/* Skeleton animation */
+@keyframes shimmer-anim {
+  0% {
+    transform: translate3d(0, 0, 0);
+  }
+  100% {
+    transform: translate3d(50%, 0, 0);
+  }
+}
+
+.skeleton-shimmer {
+  width: 200%;
```

Comment:
⛏ It's always better to organize properties by alphabetic order, it's easier to find what you want

Reply:
Yep! Updated it :)